CalibTracker/SiStripHitEfficiency/plugins/SiStripHitEffFromCalibTree.cc

```diff
@@ -1006,8 +1006,8 @@ void SiStripHitEffFromCalibTree::makeTKMap(bool autoTagging = false) {
   tkmapnum = new TrackerMap(" Detector numerator   ");
   tkmapden = new TrackerMap(" Detector denominator ");
 
-  double myeff, mynum, myden;
-  double eff_limit = 0;
+  double myeff, mynum, myden, myeff_up;
+  double layer_min_eff = 0;
```

Comment:
could these be `float`-s?

Reply:
what's the advantage?

Reply:
That's not persisted, it's an internal of the algo.

Reply:
OK, then let's keep it as it was.